### DISCUSSION TITLE
How to display the " Hello this is chaskiq support team" content in chat app navebar

### DISCUSSION THREAD

**Discussion by VinothGox** [chaskiq/chaskiq#636] [Q&A] [answered]
Hi,

How to display the Hello content in chaskiq chat app application.

After setup, the chat app does not show the content. only display the logo.
![2963ea30-215c-4084-b2aa-bb1e4ebcaacc](https://user-images.githubusercontent.com/76249948/153429541-43732289-b9cc-4f79-9b20-bfd005395f22.jpg)
![Screenshot from 2022-02-10 20-06-07](https://user-images.githubusercontent.com/76249948/153429788-1acb4af3-406d-4522-980c-ceb8b2fcf8f1.png)

**Comment by VinothGox** [comment]
Hello, content and tagline are not displayed in our app.  Any need to set up please let me know.  past two days I can't resolve the problem.

**Comment by michelson** [comment, marked as answer]
Look for the translations tab in settings

**Reply by VinothGox** [reply]
Thanks
